game/core/src/me/lihq/game/screen/NavigationScreen.java

```diff
@@ -333,11 +333,21 @@ public void updateTiledMapRenderer()
         this.currentNPCS = game.getNPCS(game.player.getRoom());
     }
 
+    public boolean getChangeMap()
+    {
+        return(changeMap);
```

Comment:
That's my bad. I told him to add a getter because he wanted access to a private variable. I don't really see the problem but stackoverflow disagrees